versioneer.py

```diff
@@ -1659,7 +1730,7 @@ def render_git_describe(pieces):
     return rendered
 
 
-def render_git_describe_long(pieces):
+def render_git_describe_long(pieces: Dict[str, Any]) -> str:
     """TAG-DISTANCE-gHEX[-dirty].
```

Comment:
Might be good to start with a description and then this
Also, One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)